torch/csrc/jit/mobile/runtime_compatibility.h

```diff
@@ -1,11 +1,20 @@
 #pragma once
 
 #include <memory>
+#include <unordered_map>
 
 namespace torch {
 namespace jit {
 
+static const int NO_SCHEMA = -1;
```

Comment:
this is probably not what I want to be doing. Anyone have a code pointer to how stuff like this is normally done in torch?

Reply:
Internally I'm using c10::optional<int>. When serializing it I use -1.